pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/BrokerStats.java

```diff
@@ -71,6 +75,21 @@
     }
 
     @GET
+    @Path("/metrics-prometheus")
+    @ApiOperation(value = "Gets the metrics in Prometheus format", notes = "")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission") })
+    @Produces(MediaType.TEXT_PLAIN)
+    public void getPrometheusMetrics(@Suspended AsyncResponse asyncResponse) throws Exception {
+        // Ensure super user access only
+        validateSuperUserAccess();
+
+        pulsar().getExecutor().execute(safeRun(() -> {
+            StreamingOutput stream = out -> new PrometheusMetricsGenerator(pulsar()).generate(out);
```

Comment:
if `pulsar().getExecutor()` has 20 threads in pool so, if all of them keeps `AggregatedNamespaceStats` as thread-local then does it make sense to use executor with lower number of thread to reduce number of stats objects in thread-local?